src/Raven.Studio/typescript/models/resources/database.ts

```diff
@@ -64,6 +71,10 @@ class database {
 
             this.relevant(inMemberList || inPromotableList || inRehabList);
         }
+
+        this.databaseAccess(accessManager.default.getEffectiveDatabaseAccessLevel(incomingCopy.Name));
+        this.databaseAccessText(accessManager.default.getAccessLevelText(this.databaseAccess()));
```

Comment:
let's convert that into computed and get right of this method, so first call accessManager to get access level, and then map to color.

Reply:
as discussed - leaving this as is